lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -23,7 +215,10 @@ public class PuppetPSHost : PSHost
     public override Guid InstanceId { get { return _hostId; } }
     public override string Name { get { return "PuppetPSHost"; } }
     public override Version Version { get { return new Version(1, 0); } }
```

Comment:
Taking this even further, we will have PSHost version 1.1, in Puppet PowerShell module 2.0.0, with PowersShell versions 2.0, 3.0, 4.0, and 5.0. Should the PSHost version match the module version to reduce the amount of differences?